src/UAEventEmitter.ts

```diff
@@ -27,12 +28,24 @@ export class UAEventEmitter {
         }
       );
     }
-    
-    InteractionManager.runAfterInteractions(() => {
+
+    if (Platform.OS === 'ios' && AppState.currentState !== 'active') {
+      AppState.addEventListener('change', (nextAppState) => {
+        if (nextAppState === 'active' && !this.iosListenerInitialized) {
```

Comment:
This will break content-available pushes (wake app up in the BG)

Reply:
You're right. I will fix that.